pkg/cmd/label/remove.go

```diff
@@ -0,0 +1,81 @@
+package label
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/cli/cli/v2/api"
+	"github.com/cli/cli/v2/internal/ghrepo"
+	"github.com/cli/cli/v2/pkg/cmdutil"
+	"github.com/cli/cli/v2/pkg/iostreams"
+	"github.com/spf13/cobra"
+)
+
+type removeOptions struct {
```

Comment:
I originally used "remove" because I ran `--help` on a few commands and saw it was common, but after doing a regex search on pkg/cmd I do see that there are 7 instances of "delete" and 2 (not counting this one) instances of "remove".